Description: Build without manpage.
 Switch from using ComplexProgramTarget() to ComplexProgramTargetNoMan() in
 IMakefile.  Otherwise, build fails with:
   "*** No rule to make target `wmcdplay.man', needed by `wmcdplay._man'."
 .
 Patch created for 1.0beta1-8.  Converted to quilt for 1.0beta-12.
Bug-Debian: http://bugs.debian.org/123732
Author: Anthony Wong <ypwong@debian.org>
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2014-07-29

--- a/Imakefile
+++ b/Imakefile
@@ -5,4 +5,4 @@
 SRCS = wmcdplay.cc
 OBJS = wmcdplay.o
 
-ComplexProgramTarget(wmcdplay)
+ComplexProgramTargetNoMan(wmcdplay)
